examples/tutorials/audio_feature_extractions_tutorial.py

```diff
@@ -34,131 +24,51 @@
 print(torchaudio.__version__)
 
 ######################################################################
-# Preparing data and utility functions (skip this section)
-# --------------------------------------------------------
+# 1. Preparation
+# --------------
 #
-
-# @title Prepare data and utility functions. {display-mode: "form"}
-# @markdown
-# @markdown You do not need to look into this cell.
-# @markdown Just execute once and you are good to go.
-# @markdown
-# @markdown In this tutorial, we will use a speech data from [VOiCES dataset](https://iqtlabs.github.io/voices/),
-# @markdown which is licensed under Creative Commos BY 4.0.
-
-# -------------------------------------------------------------------------------
-# Preparation of data and helper functions.
-# -------------------------------------------------------------------------------
-
-import os
-
+# .. note::
+#
+#    When running this tutorial in Google Colab, install the required packages
+#
+#    .. code::
+#
+#       !pip install torchaudio librosa
```

Comment:
torchaudio is pre-installed in Colab, is it necessary to force install it?

Reply:
No it is a residual from the past. thanks for pointing.

```suggestion
#       !pip install librosa
```